packages/fast-components-react-base/src/listbox/README.md

```diff
@@ -0,0 +1,10 @@
+## Listbox
+A generic menu component implementing the [listbox](https://www.w3.org/TR/wai-aria-1.1/#listbox) role and the keyboard navigation aligning to that role. The *listbox* will accept any child elements but only children with role of `"option"` will receive focus.
+
+### Usage
+To manage document focus the *listbox* assumes that all rendered children that have the role `"option"` and not decorated with aria-disabled are focusable html elements. It also expects that these elements are passed as *direct* children. Wrapping all menu items inside of a `React.Fragment` (or some other single element) will throw off the internal focus tracking of the *listbox*. Additionally, the *listbox* expects that all menu item children implement the `tabIndex` property and apply that property to the underlying DOM.
```

Comment:
> Additionally, the *listbox* expects that all menu item children implement the `tabIndex` property and apply that property to the underlying DOM.

We are setting tabIndex and managing focus; is this actually accurate?

Reply:
Same wording and underlying functionality as context menu.

Reply:
OK - Is it accurate? If it is, that's fine; it seems incorrect as we are handling it. If this is incorrect, should we still commit this?

Reply:
I see. What we are actually saying is that the component manages the tabindex of children so children should not interfere with that. Yes?

Reply:
"Additionally, the *listbox* component sets the `tabIndex` property of its children as it manages focus, child components should not set tabindex independently."?